cmd/vulcan-dkim/main.go

```diff
@@ -216,7 +216,7 @@ type options struct {
 }
 
 func main() {
-	run := func(ctx context.Context, target string, optJSON string, state state.State) (err error) {
+	run := func(ctx context.Context, target, assetType, optJSON string, state state.State) (err error) {
```

Comment:
Shouldn't we check if it's reachable?

Reply:
Done, I think I skipped this one when we did not have a verification in place for DomainNames.